.github/workflows/root-ci.yml

```diff
@@ -414,6 +430,30 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref_name }}
+      
+      - name: Install Google Chrome
+        shell: bash
+        run: |
+          if [[ "${{ matrix.image }}" == "fedora39" ]]; then
+            dnf install -y dnf-plugins-core
+            dnf install fedora-workstation-repositories -y
+            dnf config-manager --set-enabled google-chrome
+            dnf install google-chrome-stable -y
+          elif [[ "${{ matrix.image }}" == "alma8" ]] || [[ "${{ matrix.image }}" == "alma9" ]] || [[ "${{ matrix.image }}" == "alma9-clang" ]]; then
+            dnf install -y wget
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
+            dnf install -y ./google-chrome-stable_current_x86_64.rpm
+          elif [[ "${{ matrix.image }}" == "ubuntu20" ]] || [[ "${{ matrix.image }}" == "ubuntu22" ]] || [[ "${{ matrix.image }}" == "ubuntu2404" ]]; then
+            apt update
+            apt install -y wget
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            apt install -y ./google-chrome-stable_current_amd64.deb
+          elif [[ "${{ matrix.image }}" == "debian125" ]]; then
+            apt update
+            apt install -y wget
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            apt install -y ./google-chrome-stable_current_amd64.deb
+          fi
 
```

Comment:
Why `Google Chrome` is used here? Why not to use `chromium` - it typically provided in all distributions.

Reply:
Ok I'll change that

Reply:
For the moment I see only installation of chromium - where it used then?

Reply:
I need it to run the new graphic svg test in roottest. https://github.com/root-project/roottest/pull/1179

Reply:
I see now. Does it works with `roottest` now?

Reply:
For MAC the chrome/chromium installation is still missing. I think someone with authority has to do it.
The new svg tests with headless chrome browser are not hanging on MAC, at least on my machine. So after the installation it should be posisble to enabled all tests here.

For Linux the chromium installation is implemented, but here the headless browser is sometimes hanging.
That's why only 15 new svg tests are enabled. This is also the reason why I am currently differentiating between MAC and Linux platforms in my CMakeLists in roottest/graphics.

Yes in general there is a set of all test types, which works on Linux and should also work on MAC after the chrome/chromium installation in the ROOT CI

Reply:
Ill probably have to regenerate some references, but that's it.